### PR TITLE
Fixes #12748: Adding missings informations for Windows plugin installation

### PR DESCRIPTION
**Pull request by ncharles**
https://www.rudder-project.org/redmine/issues/12748

### CODE REVIEW

**Comment by ncharles** [comment]
Commit modified

File: 95_windows/20_agent_installation.txt [hunk 1]
```diff
@@ -2,6 +2,37 @@
...
 
 The installation and upgrade processes are exactly the same.
+
+===== Supported version of Microsoft Windows
+
+The Rudder agent needs *PowerShell 4* or later, which is built-in on:
+
+ * Windows Server 2012 R2 and later
+
+PowerShell 4 may also be installed on the following platforms, following the this procedure: https://social.technet.microsoft.com/wiki/contents/articles/20623.step-by-step-upgrading-the-powershell-version-4-on-2008-r2.aspx
```
Comment (fanf) [inline 95_windows/20_agent_installation.txt]: following the this/following this

File: 95_windows/20_agent_installation.txt [hunk 2]
```diff
@@ -2,6 +2,37 @@
...
+ * Windows Server 2012 R2 and later
+
+PowerShell 4 may also be installed on the following platforms, following the this procedure: https://social.technet.microsoft.com/wiki/contents/articles/20623.step-by-step-upgrading-the-powershell-version-4-on-2008-r2.aspx 
+
+ * Windows Server 2008 R2
+ * Windows Server 2012
+
+===== Desktop version of Microsoft Windows
+
+There is no official support of desktop versions of Microsoft Windows. However, the agent can be installed on the following platform:
```
Comment (fanf) [inline 95_windows/20_agent_installation.txt]: There is no official support of /Rudder agent/ on desktop versions of Microsoft Windows.

**Comment by ncharles** [comment]
Commit modified

**Pull request merged**